### PR TITLE
Fix / a11y readonly textfields & secret creditcard number and security code

### PR DESCRIPTION
**Pull request by langemike**
See original PR (which is closed to deleted original branch): https://github.com/Videodock/ott-web-app/pull/16

Requesting new reviews, because I rewrote the e2e tests and I would like to know if you agree!

### CODE REVIEW

File: packages/ui-react/src/components/TextField/TextField.test.tsx [hunk 1]
```diff
@@ -10,12 +10,26 @@ describe('<TextField>', () => {
     expect(container).toMatchSnapshot();
   });
 
+  test('renders and matches readonly snapshot', () => {
+    const { container } = render(<TextField label="Label" placeholder="Placeholder" name="name" value="" onChange={vi.fn()} readOnly={true} />);
```
Comment (royschut) [inline packages/ui-react/src/components/TextField/TextField.test.tsx:14]: :) 
```suggestion
    const { container } = render(<TextField label="Label" placeholder="Placeholder" name="name" value="" onChange={vi.fn()} readOnly />);
```

**Review by royschut — APPROVED** [review]
Nice!

**Review by ChristiaanScheermeijer — APPROVED** [review]
lgtm!

**Comment by royschut** [comment]
Let's merge it a.s.a.p.! (Don't forget to squash) @langemike

**Review by MelissaDTH — APPROVED** [review]

**Pull request merged**